src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,67 @@
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { selectTodo } from '../../features/currentTodo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const dispatch = useAppDispatch();
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+  const ID_COMPARISON = todo.id === selectedTodo?.id;
+
+  const handleClick = () => {
+    dispatch(selectTodo(todo));
+  };
+
+  return (
+    <tr
+      data-cy="todo"
+      className={classNames({
+        'has-background-info-light': ID_COMPARISON,
+      })}
+    >
+      <td className="is-vcentered">{todo.id}</td>
+      {!todo.completed ? (
+        <td className="is-vcentered"> </td>
+      ) : (
+        <td className="is-vcentered">
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        </td>
```

Comment:
You have `<td className="is-vcentered"> </td>` in both cases only a content is different